cmd/livepeer/livepeer.go

```diff
@@ -429,7 +429,12 @@ func main() {
 		go senderWatcher.Watch()
 		defer senderWatcher.Stop()
 
-		orchWatcher, err := watchers.NewOrchestratorWatcher(addrMap["BondingManager"], blockWatcher, dbh, n.Eth, roundsWatcher)
+		// When in orchestrator mode only process TranscoderActivated/TranscoderDeactivated events for O
+		var addressFilter []ethcommon.Address
```

Comment:
Don't need to filter addresses in `OrchestratorWatcher` anymore in this PR.

Reply:
Mind that orchestratorWatcher is also started for O and will process all activation and deactivation events for no reason.

Reply:
> Mind that orchestratorWatcher is also started for O and will process all activation and deactivation events for no reason.

This all happens in the background so I don't think this is an immediate concern.